MusicNotationKit/MusicNotationKit/Tone.swift

```diff
@@ -18,3 +18,13 @@ public struct Tone {
 		self.octave = octave
 	}
 }
+
+extension Tone: CustomDebugStringConvertible {
+	public var debugDescription: String {
+		var result = String(format:"%@%d", String(self.noteLetter), self.octave.rawValue)
+		if (self.accidental != nil ) {
+			result = String(format: "[%@%@]", result, self.accidental!.rawValue)
+		}
```

Comment:
As before, use string interpolation and the nil coalescing operator and you shouldn't need the temp variable.